drivers/storage/vbox/vbox.go

```diff
@@ -27,7 +27,7 @@ func registerConfig() {
 	r.Key(gofig.String, "", volumePath, "", "virtualbox.volumePath")
 	r.Key(gofig.String, "", "", "", "virtualbox.localMachineNameOrId")
 	r.Key(gofig.Bool, "", false, "", "virtualbox.tls")
-	r.Key(gofig.String, "", "SATAController", "", "virtualbox.controllerName")
```

Comment:
Let's change this back to the existing setting. All of our examples that don't specify the SATA Controller expect this name. Since it is really an abritrary name, I wouldn't change how we default it at this point.